src/common/wincmn.cpp

```diff
@@ -2871,9 +2894,16 @@ wxWindowBase::FromDIP(const wxSize& sz, const wxWindowBase* WXUNUSED(w))
 
 /* static */
 wxSize
-wxWindowBase::ToDIP(const wxSize& sz, const wxWindowBase* WXUNUSED(w))
+wxWindowBase::ToDIP(const wxSize& sz, const wxWindowBase* w)
 {
-    const wxSize dpi = wxScreenDC().GetPPI();
+    wxSize dpi;
```

Comment:
This, admittedly trivial, fragment is repeated thrice, so I think it would make sense to add some
```
bool GetDPI(wxSize& dpi, wxWindow *w);
```
helper that would do it only once. What do you think?

Name is, as always, subject for discussion/your choice.